style/base.css

```diff
@@ -0,0 +1,489 @@
+    padding: 0 2rem;
+}
+
+#solid-organizations>div>ul {
+    list-style: none;
+}
+
+#about section#features ul li,
+#hosted-pod-services>div>ul li,
+#solid-organizations>div>ul li,
+#where-to-collaborate,
+#events>div>ul {
+    background-color: #fff;
+    border-radius: 0.5rem;
+    padding: 2rem;
+    color: #000;
+}
+
+#events li {
+    margin: 3rem 0;
+}
+
+#events details summary {
+    font-size: 1.5rem;
+    list-style: none;
+    cursor: pointer;
+    position: relative;
+    padding-right: 20px;
+    margin-bottom: 1rem;
+}
+
+#events summary::marker {
+    content: none;
+}
+
+#events summary::after {
+    content: '\2303';
+    font-size: 2.5rem;
+    position: absolute;
+    top: calc(50% - 0.2rem);
+    transform: translateY(calc(-50% - 0.5rem)) rotate(180deg);
+}
+
+#events [open] summary::after {
+    transform: translateY(calc(-50% + 0.5rem)) rotate(0deg);
+}
+
+#where-to-collaborate>div>ul,
+#events>div>ul>li>details>ul {
+    list-style: none;
+}
+
+#events>div>ul>li>details>ul li {
+    border: 1px solid #979797;
+    border-radius: 0.5rem;
+}
+
+#events>div>ul>li>details>ul li dt {
+    display: none;
+}
+
+#events details ul {
+    padding: 0;
+}
+
+#events details dl {
+    padding: 1em;
+}
+
+#events details dl dt,
+#events details dl dd {
+    padding: 0;
+    margin: 0
+}
+
+#events details li {
+    margin-top: 0;
+}
+
+#solutions-using-solid-technology>div>ul,
+#hosted-pod-services>div>ul,
+#solid-organizations>div>ul,
+#features ul {
+    list-style: none;
+}
+
+#hosted-pod-services>div>ul li,
+#solid-organizations>div>ul li {
+    background-color: #FEFEFE;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
+    margin-bottom: 2rem;
+}
+
+#about section#features ul li a,
+#where-to-collaborate ul li a {
+    background-color: #7B4DFF;
+    padding: 0.8rem 1.5rem;
+    color: #fff;
+    text-decoration: none;
+    text-wrap: nowrap;
+}
+
+#developer-tutorials {
+    margin: 0;
+}
+
+
+/* Desktop styles */
+@media (min-width: 767px) {
+    header {
+        flex-direction: row;
+        align-items: center;
+    }
+
+    header a,
+    footer a {
+        text-wrap: nowrap;
+    }
+
+    nav ul {
+        display: flex;
+        align-self: flex-end;
+    }
+
+    main {
+        padding: 6rem;
+    }
+
+    main>article {
+        display: flex;
+        flex-direction: column;
+    }
+
+    main>article>h1,
+    main>article>p {
+        color: #fff;
+        font-weight: 400;
+        padding: 0 6rem;
+    }
+
+    #homepage main>article>div>section>div>ul,
+    #developer-tutorials {
+        border-radius: 30px;
+    }
+    
+    #solutions-using-solid-technology h2 {
+        padding: 0 6rem;
+    }
+
+    #homepage main>article>div>section>div>ul {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(2, 1fr);
+        grid-column-gap: 0;
+        grid-row-gap: 0;
+    }
+
+    #homepage main>article>div>section>div>ul li {
+        display: grid;
```

Comment:
Do you prefer using grid here? My first instinct would be to use one horizontal flex for the image and text box, and then a vertical flex for the text, and then wrap the horizontal flex on mobile. It looks like you got it working fine, it was just more difficult for me to figure out with grid